solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -14,80 +14,62 @@ Following are the minimum system specifications required to set up the K8s clust
 This section enlists the commands to deploy the CORTX Stack on K8s cluster. 
 
 ### Make sure your VM(virtual machine) has following drives available on it:
-
 ```
-# ls /dev/sd*
+ls /dev/sd*
 
 /dev/sda  /dev/sda1  /dev/sda2  /dev/sdb  /dev/sdc  /dev/sdd  /dev/sde  /dev/sdf  /dev/sdg  /dev/sdh  /dev/sdi
 ```
 
 ### You must have Git installed on your system.
 You can use following command for RedHat family OS to install the Git.
-
 ```
-# yum install git -y
+yum install git -y
 ```
 
 ### SELinux should be disabled
-
 -  Use the following command to check status of SELinux.
 ```
-# sestatus
+sestatus
 ```
 -  If SELinux is enabled, run the following command to disable the SELinux.
-
 ```
-# sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
+sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
 ```
-
 -  Once above command runs successfully, reboot your system.
-
 ```
-# reboot
+reboot
 ```   
 
-### Note
- 1. All the nodes should be reachable over SSH with root user.
+**Note:**
+ 1. All the EC2 nodes should be reachable over SSH
```

Comment:
Please remove EC2. These instructions are generic for VM's created on any platform

Reply:
done